src/main/java/io/cdap/plugin/salesforce/plugin/source/streaming/SalesforcePushTopicListener.java

```diff
@@ -176,7 +177,7 @@ public void createSalesforceListener() throws Exception {
             }
           } else {
             throw new RuntimeException(String.format("Error in meta connect, errorMessage: %s Advice: %s", error,
-                                                     advice));
+                    advice));
```

Comment:
Formatting looks incorrect here.

Reply:
Corrected it. e130c2d587656a458efc1c7b64df2f795710543c